crates/swc/src/builder.rs

```diff
@@ -270,7 +270,20 @@ impl<'a, 'b, P: swc_ecma_visit::Fold> PassBuilder<'a, 'b, P> {
             ),
             compat::reserved_words::reserved_words(),
             Optional::new(export_namespace_from(), need_interop_analysis),
-            Optional::new(helpers::inject_helpers(), self.inject_helpers),
+            Optional::new(
+                helpers::inject_helpers(match module {
+                    Option::None => swc_ecma_transforms_base::helpers::ModuleType::Es6, /* TODO: is this assumption correct? */
```

Comment:
@kdy1 is it correct to assume ESM when we do not have a module config?

Reply:
Yes. It's default behavior.